utils/save_filter_data.py

```diff
@@ -0,0 +1,9 @@
+import csv
+from pathlib import Path
+
+
+def save_filter_data(total, filtered, thresh, output_path):
```

Comment:
This is more about saving settings and aggregated data?
```suggestion
def save_segmentation_info(total, filtered, thresh, output_path):
```

Reply:
Not sure if it makes too much sense to have a csv file with only one entry.
Store info about all image segmentations in a single json/csv file.

Depends on what you want to do with the info later.

Reply:
Changed it now so all info is tored in a single json file. The thought was to see how useful the excluding etc. is.